### PR TITLE
[#1455] aggregate memory stat of agent data by using flink

### PR DESCRIPTION
**Pull request by minwoo-jung**

### CODE REVIEW

**Comment by minwoo-jung** [comment]
please review @Xylus

File: collector/src/test/java/com/navercorp/pinpoint/collector/mapper/thrift/stat/TFAgentStatMappterTest.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author minwoo.jung
+ */
+public class TFAgentStatMappterTest {
```
Comment (Xylus) [inline collector/src/test/java/com/navercorp/pinpoint/collector/mapper/thrift/stat/TFAgentStatMappterTest.java:33]: typo, `TFAgentStatMappterTest` -> `TFAgentStatMapperTest`?

File: collector/src/main/java/com/navercorp/pinpoint/collector/mapper/thrift/stat/TFAgentStatMappter.java [hunk 2]
```diff
@@ -17,42 +17,64 @@
...
+import com.navercorp.pinpoint.thrift.dto.flink.TFJvmGc;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
 
 /**
  * @author minwoo.jung
  */
 public class TFAgentStatMappter {
```
Comment (Xylus) [inline collector/src/main/java/com/navercorp/pinpoint/collector/mapper/thrift/stat/TFAgentStatMappter.java:29]: typo, `TFAgentStatMappter` -> `TFAgentStatMapper`?

File: collector/src/test/java/com/navercorp/pinpoint/collector/mapper/thrift/stat/TFAgentStatMappterTest.java (hunk 1 above)
Comment (minwoo-jung) [reply collector/src/test/java/com/navercorp/pinpoint/collector/mapper/thrift/stat/TFAgentStatMappterTest.java:33]: OK. I will fix it.

File: collector/src/main/java/com/navercorp/pinpoint/collector/mapper/thrift/stat/TFAgentStatMappter.java (hunk 2 above)
Comment (minwoo-jung) [reply collector/src/main/java/com/navercorp/pinpoint/collector/mapper/thrift/stat/TFAgentStatMappter.java:29]: OK. I will fix it.

**Pull request merged**